extensions/wikia/PortableInfobox/controllers/ApiQueryPortableInfobox.class.php

```diff
@@ -38,6 +38,14 @@ protected function runOnPageSet( ApiPageSet $pageSet ) {
 					$pageSet->getResult()->addValue(
 						[ 'query', 'pages', $id, 'infoboxes', $count ], 'sourcelabels', $sl
 					);
+
+					// TODO is this even needed? Seems to work without this code
+					if ( isset( $infobox[ 'sourcetypes' ] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "sourcetypes" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)